pkg/k8s/forward/manager.go

```diff
@@ -118,8 +118,13 @@ func (p *PortForwardManager) Add(f model.Forward) error {
 	return nil
 }
 
+// StartGlobalForwarding is not implemented
+func (*PortForwardManager) StartGlobalForwarding() error {
+	return fmt.Errorf("not implemented")
+}
+
 // AddReverse is not implemented
-func (p *PortForwardManager) AddReverse(_ model.Reverse) error {
+func (*PortForwardManager) AddReverse(_ model.Reverse) error {
```

Comment:
Why does this exist now? As far as I know we aren't using any imported code for this and therefore also control the interface so we shouldn't need to add a stub to satisfy an interface. It seems like we should create a different interface and possibly combine interfaces.

Reply:
This and your suggestion regarding duplicate code I think could be addressed in this issue #2770. WDYT? If so, I can add some more information in the issue to complete the refactor properly.

Reply:
We're already making big changes and this is a new feature. I would rather not add to technical debt as part of a new feature in cli at this point. I think using the interface and the "ifs" on global vs. non-global is a non-ideal implementation that adds to brittleness and testing difficulty. I'd rather have clear code paths for global vs non-global that are handled by struct initilization and types then conditionals.